src/Symfony/Bundle/WebProfilerBundle/Controller/ProfilerController.php

```diff
@@ -162,6 +162,28 @@ public function toolbarAction(Request $request, ?string $token = null): Response
         ]);
     }
 
+    /**
+     * Renders the Web Debug Toolbar stylesheet.
+     *
+     * @throws NotFoundHttpException
+     */
+    public function toolbarStylesheetAction(): Response
+    {
+        $this->denyAccessIfProfilerDisabled();
+
+        $this->profiler->disable();
```

Comment:
`denyAccessIfProfilerDisabled` already disables the profiling for the current request

Reply:
Thank you! Updated

Reply:
I realize someone already did the same mistake in the same file .... me :) 

Should i fix in the same PR or create another one ? (in the `fontAction`)